schemas/com.snowplowanalytics.factotum/job_update/jsonschema/1-0-0

```diff
@@ -0,0 +1,109 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"self": {
+		"vendor": "com.snowplowanalytics.factotum",
+		"name": "job_update",
+		"version": "1-0-0",
+		"format": "jsonschema"
+	},
+	"type": "object",
+	"properties": {
+		"schema": {
+			"type": "string"
+		},
+		"data": {
+			"type": "object",
+			"properties": {
+				"jobName": {
+					"type": "string"
+				},
+				"runReference": {
+					"type": "string"
+				},
+				"factfile": {
+					"type": "string"
+				},
+				"factotumVersion": {
+					"type": "string"
+				},
```

Comment:
I am tempted for us to use an application_context instead of individual properties like factotumVersion